.clang-format

```diff
@@ -4,3 +4,4 @@ SortIncludes: false
 AlignConsecutiveAssignments: true
 AllowShortCaseLabelsOnASingleLine: true
 AllowShortIfStatementsOnASingleLine: true
+ColumnLimit: 100
```

Comment:
You should bring this up to a stand-up meeting, this would require reformatting every file in the repo which might be a bigger change than you anticipate?

Reply:
@lucbv ,

> this would require reformatting every file in the repo

I was worried about that too but then I noticed that the AT format check passed this PR. I think it only checks files that are modified by the PR, so we could incrementally update the formatting of the files as they are touched, no mass reformat is necessary.

Reply:
Sure but what if someone also decides that they like 105 better in their next PR? I think we should have a discussion and decide was is appropriate and then keep it fixed. For instance is there a good reason to go from the current length (80 I guess) to 100?

Reply:
I would advocate for 120 here. That being said, the original decision, IIRC, was to follow the same formatting as Kokkos Core in: https://github.com/kokkos/kokkos/blob/develop/.clang-format. I see pros and cons with diverging from the Kokkos Core formatting but am generally in favor of 120 character column limits. 80 characters is too short these days and are an artifact from when terminals were limited to such character lengths.

Reply:
In general a lot of our functions have longer names and argument lists than stuff in core... I like 120 as well

Reply:
(if we're gonna change it)

Reply:
I would be fine with 120 as well.